language/move-lang/tests/move_check/parser/byte_string_invalid_escaped_sequence.exp

```diff
@@ -0,0 +1,8 @@
+error: 
+
+   ┌── tests/move_check/parser/byte_string_invalid_escaped_sequence.move:3:14 ───
+   │
+ 3 │         b"lib\zra"
+   │              ^ invalid escape sequence: '\z'
```

Comment:
The loc reported here should also include the "z"

Reply:
Done.